.travis/osx..script.sh

```diff
@@ -1,8 +1,9 @@
 #!/usr/bin/env bash
 
-if [ $QT5 ]; then
+if [ "$QT5" ]; then
         # Workaround; No FindQt5.cmake module exists
-        export CMAKE_PREFIX_PATH="$(brew --prefix qt55)"
+        CMAKE_PREFIX_PATH="$(brew --prefix qt55)"
+        export CMAKE_PREFIX_PATH
 fi
 
-cmake $CMAKE_FLAGS -DUSE_WERROR=OFF ..
+cmake "$CMAKE_FLAGS" -DUSE_WERROR=OFF ..
```

Comment:
Is this correct?  Won't be run into same problems as `apt-get` ? I'd expect `cmake` to actually want the spaces although it may be intelligent enough to take it this way, I don't think it's right.